### PR TITLE
merge back to master: update doc index

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: docs/en_US/builtin_assessor.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
 Builtin-Assessors
 =================
 
+In order to save our computing resources, NNI supports an early stop policy and creates **Assessor** to finish this job.
+
+Assessor receives the intermediate result from Trial and decides whether the Trial should be killed by specific algorithm. Once the Trial experiment meets the early stop conditions(which means assessor is pessimistic about the final results), the assessor will kill the trial and the status of trial will be `"EARLY_STOPPED"`.
```
Comment (scarlett2018) [inline docs/en_US/builtin_assessor.rst]: > Assessor receives the intermediate result

Assessor receives the intermediate results

>the status of trial will be `"EARLY_STOPPED"`.

the status of trial will change to `"EARLY_STOPPED"`.

File: docs/en_US/builtin_assessor.rst [hunk 2]
```diff
@@ -1,6 +1,16 @@
 Builtin-Assessors
 =================
 
+In order to save our computing resources, NNI supports an early stop policy and creates **Assessor** to finish this job.
```
Comment (scarlett2018) [inline docs/en_US/builtin_assessor.rst]: Sometimes to save computing resources, we need to stop the under performance trials earlier. NNI supports early stop mechanism with built-in as well as self defined **Assessor** in the way similar to Tuners.

File: docs/en_US/builtin_assessor.rst [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+
+Assessor receives the intermediate result from Trial and decides whether the Trial should be killed by specific algorithm. Once the Trial experiment meets the early stop conditions(which means assessor is pessimistic about the final results), the assessor will kill the trial and the status of trial will be `"EARLY_STOPPED"`.
+
+Here is an experimental result of MNIST after using 'Curvefitting' Assessor in 'maximize' mode, you can see that assessor successfully **early stopped** many trials with bad hyperparameters in advance. If you use assessor, we may get better hyperparameters under the same computing resources.
```
Comment (scarlett2018) [inline docs/en_US/builtin_assessor.rst]: >bad hyperparameters 

under performed hyperparameters 

> If you use assessor, we may get better hyperparameters under the same computing resources.

With Assessor enabled, we are more likely to get better hyperparameters under the same computing resources or time spent restrictions.

File: docs/en_US/hyperparameter_tune.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
+#############################
+Auto (Hyper-parameter) Tuning
+#############################
+
+Auto tuning is one of the key features provided by NNI, a main application scenario is
+hyper-parameter tuning. Trial code is the one to be tuned, we provide a lot of popular
+auto tuning algorithms (called Tuner), and some early stop algorithms (called Assessor).
+NNI supports running trial on various training platforms, for example, on a local machine,
```
Comment (scarlett2018) [inline docs/en_US/hyperparameter_tune.rst]: >NNI supports running trial on various training platforms
These trial codes can be distributed onto various training platforms

File: docs/en_US/NAS/DARTS.md [hunk 5]
```diff
@@ -43,8 +43,10 @@ python3 retrain.py --arc-checkpoint ./checkpoints/epoch_49.json
...
 
-    .. automethod:: __init__
-
 ..  autoclass:: nni.nas.pytorch.darts.DartsMutator
     :members:
 ```
+
+## Limitations
+
+* DARTS doesn't support DataParallel and needs to be customized in order to support DistributedDataParallel.
```
[the remark this answers is not in the document]
Comment (QuanluZhang) [reply docs/en_US/NAS/DARTS.md]: will update this part in the next pr

**Pull request merged**